test/Swashbuckle.AspNetCore.IntegrationTests/ReDocIntegrationTests.cs

```diff
@@ -162,4 +164,43 @@ public void ReDocOptions_Extensions()
         Assert.True(options.ConfigObject.SortPropsAlphabetically);
         Assert.True(options.ConfigObject.UntrustedSpec);
     }
+
+    [Fact]
+    public async Task ReDocMiddleware_Returns_ExpectedAssetContents()
+    {
+        var site = new TestSite(typeof(ReDocApp.Startup));
+        using var client = site.BuildClient();
+
+        var diskFile = "redoc/bundles/redoc.standalone.js";
+        var diskFileName = Path.GetFileName("redoc/bundles/redoc.standalone.js");
+
+        using var htmlResponse = await client.GetAsync("/Api-Docs/redoc.standalone.js", TestContext.Current.CancellationToken);
+        Assert.Equal(HttpStatusCode.OK, htmlResponse.StatusCode);
+
+        using var stream = await htmlResponse.Content.ReadAsStreamAsync(TestContext.Current.CancellationToken);
+        using var diskFileStream = File.OpenRead(diskFile);
+
+        Assert.Equal(MD5.HashData(diskFileStream), MD5.HashData(stream));
```

Comment:
Please use SHA1 - it avoids tools raising warnings related to using insecure hash algorithms. Even if this is only test code, it's still noise we can avoid.